src/screens/LearningScreen/components/FlashCardContainer.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect } from 'react';
+import { View, Button, Text } from 'react-native';
+
+import { getUserData } from '../../../utils/storage';
+import {
+  getRandomWord,
+  handleCorrectOrIncorrect,
+  getNewWordLevel,
+  LIMITS,
+  LEVELS,
+} from '../../../utils/spacing';
+import FlashCard from './FlashCard';
+
+const handleRevealButtonPress = (setRevealed) => {
+  setRevealed((flipped) => !flipped);
+};
+
+const handleExpressionsButtonPress = (setExpanded) => {
+  setExpanded((isExpanded) => !isExpanded);
+};
+
+const FlashCardContainer = () => {
+  const [list, setList] = useState({});
+  const [wordInfo, setWordInfo] = useState({});
+  const [isExpanded, setExpanded] = useState(false);
```

Comment:
Like we discussed, I would definitely look at `useReducer` to manage all this interrelated state.